.github/workflows/remove-wheels.yml

```diff
@@ -0,0 +1,78 @@
+name: Remove old wheels
+
+on:
+  # Run daily at 1:23 UTC
+  schedule:
+  - cron:  '23 1 * * *'
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+# Needed for micromamba pickup
+defaults:
+  run:
+    shell: bash -l {0}
+
+env:
+  N_LATEST_UPLOADS: 5
+  ANACONDA_USER: "scientific-python-nightly-wheels"
+
+jobs:
+  remove:
+
+    runs-on: ubuntu-latest
+    environment:
+      name: remove-old-wheels
+
+    steps:
+      - name: Install micromamba and anaconda-client
+        uses: mamba-org/setup-micromamba@v1
+        with:
+          environment-name: remove-wheels
+          create-args: >-
+            python=3.11
+            anaconda-client=1.11.2
+
+      - name: Show environment
+        run: env
+
+      - name: Show CLI API info
```

Comment:
Not versions, but if someone is looking to the logs for debugging information it can be useful to understand what the `anaconda-client` CLI API is given that there documentation that exits for it (https://docs.anaconda.org/free/anacondaorg/commandreference/) is literally just the same thing. :/